PWGHF/TableProducer/candidateSelectorDstar.cxx

```diff
@@ -0,0 +1,285 @@
+// In this version of task I am trying to extend the d0 table with expression columns.
+// MC matching is not done yet.
+// When I was using O2DatabasePDG: program was crashing.
+// Retest after solving debug problem in hf-track-index-skim-creator
```

Comment:
Is this still the case? (it anyway has to be moved from here to make Clang happy)
```suggestion
```